runtime/doc/news.txt

```diff
@@ -40,6 +40,8 @@ The following new APIs or features were added.
   iterators |luaref-in|.
 
 • Added |vim.keycode()| for translating keycodes in a string.
+• We now offer to show detected to news.txt (prerelease
```

Comment:
want see if we can avoid saying 'we now...' in every single news item

```suggestion
• Nvim shows news.txt on startup if it is a prerelease that has news since the previous build.
```